### PR TITLE
keep Docker images alpine based

### PR DESCRIPTION
**Pull request by Zoey2936**

### CODE REVIEW

File: docker/proxy/Dockerfile [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
+FROM --platform=${BUILDPLATFORM} golang:1.20-alpine AS builder
 ARG TARGETARCH
 ARG TARGETOS
 
 WORKDIR /workdir
 
 COPY . .
-RUN apt-get -y update && \
-    apt-get -y install protobuf-compiler && \
+RUN apk add --no-cache bash build-base protobuf && \
```
Comment (fancycode) [inline docker/proxy/Dockerfile:8]: Please also install `git` here. This is not included in the Go `-alpine` image, but is used to determine the version of the signaling proxy (would be `unknown` now).

File: docker/server/Dockerfile [hunk 2]
```diff
@@ -1,28 +1,25 @@
...
+FROM --platform=${BUILDPLATFORM} golang:1.20-alpine AS builder
 ARG TARGETARCH
 ARG TARGETOS
 
+WORKDIR /workdir
+
 COPY . .
-RUN apt-get -y update && \
-    apt-get -y install protobuf-compiler && \
+RUN apk add --no-cache bash build-base protobuf && \
```
Comment (fancycode) [inline docker/server/Dockerfile:8]: Same as above.

**Review by fancycode — CHANGES_REQUESTED** [review]

File: docker/server/Dockerfile (hunk 2 above)
Comment (Zoey2936) [reply docker/server/Dockerfile]: done

File: docker/proxy/Dockerfile (hunk 1 above)
Comment (Zoey2936) [reply docker/proxy/Dockerfile]: done

**Comment by Zoey2936** [comment]
thanks!

**Comment by fancycode** [comment]
Thanks! Assuming you need a new release for https://github.com/nextcloud/all-in-one/pull/1583?

I will wait until early next week to see if anything else comes up and will do the release then.

**Comment by Zoey2936** [comment]
> Thanks! Assuming you need a new release for [nextcloud/all-in-one#1583](https://github.com/nextcloud/all-in-one/pull/1583)?

yes

> I will wait until early next week to see if anything else comes up and will do the release then.

thanks

**Comment by Zoey2936** [comment]
> I will wait until early next week to see if anything else comes up and will do the release then.

What needs to be done for a new release?

**Comment by fancycode** [comment]
Sorry, too many other things came up and I couldn't finish the release in time.
New release is available now: https://github.com/strukturag/nextcloud-spreed-signaling/releases/tag/v1.1.2

**Comment by Zoey2936** [comment]
thanks!

**Pull request merged**